### ISSUE TITLE
Crash reporter missing in testpilotcloud 5.0.0.11375-daily20230704

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/client#10998]
```
% /Applications/testpilotcloud.app/Contents/MacOS/testpilotcloud --debug
zsh: abort      /Applications/testpilotcloud.app/Contents/MacOS/testpilotcloud --debug
```

Then crash with `Debug actions > Crash now - qFatal`

```
% /Applications/testpilotcloud.app/Contents/MacOS/testpilotcloud --version
testpilotcloud testpilotcloud 5.0.0.11375-daily20230704
511b96
Libraries Qt 6.4.3, OpenSSL 1.1.1t  7 Feb 2023
Using virtual files plugin: suffix
macos-22.5.0
```

(M1 MacBook Air)

**Comment by michaelstingl** [comment]
A new file was created:

```
% ls -l | grep testpilotcloud
-rw-r--r--   1 michaelstingl  staff    3339 Jul  4 11:28 testpilotcloud-crash.log
drwxr-xr-x  10 michaelstingl  staff     320 Jul  6 09:27 testpilotcloud-logdir
```

**Comment by HanaGemela** [comment]
Testing blocked by https://github.com/owncloud/client/issues/11039

**Comment by HanaGemela** [comment]
testpilotcloud 5.0.0.11480-daily20230717, Intel mac

```
Hanas-MBP:~ hana$ /Applications/testpilotcloud.app/Contents/MacOS/testpilotcloud --debug
Abort trap: 6
```

Apple report created instead 
<img width="1000" alt="Screenshot 2023-07-17 at 19 52 15" src="https://github.com/owncloud/client/assets/49001702/0f4e9f9b-fdc0-4fec-9bd1-9302f6438d2a">

**Comment by TheOneRing** [comment]
Internal ci issue has been fixed

**Comment by HanaGemela** [comment]
Fixed in testpilotcloud 5.0.0.11501-daily20230719
